apps/web/src/pages/Pools/AssessmentPlanBuilderPage/components/ConfirmationDialog.tsx

```diff
@@ -0,0 +1,75 @@
+import { useState } from "react";
+import TrashIcon from "@heroicons/react/24/solid/TrashIcon";
+import { useIntl } from "react-intl";
+
+import { Button, Dialog, Well } from "@gc-digital-talent/ui";
+import { formMessages } from "@gc-digital-talent/i18n";
+import { Maybe } from "@gc-digital-talent/graphql";
+
+interface ConfirmationDialogProps {
+  assessmentTitle?: Maybe<string>;
+  onRemove: () => Promise<void>;
+}
+
+const ConfirmationDialog = ({
+  assessmentTitle,
+  onRemove,
+}: ConfirmationDialogProps) => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const intl = useIntl();
+
+  const title = intl.formatMessage({
+    defaultMessage: "Delete assessment",
+    id: "+WT43U",
+    description: "Title to delete an assessment",
+  });
+
+  const handleRemove = async () => {
+    await onRemove().then(() => {
+      setIsOpen(false);
+    });
+  };
+
+  return (
+    <Dialog.Root open={isOpen} onOpenChange={setIsOpen}>
```

Comment:
Confirmation dialogs should probably be an `AlertDialog` since we require an action from the user (delete or cancel)

Reply:
Good call :+1: Fixed in [Change to alert dialog](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14177/commits/14063a6d7f3f2e73745943846901137b8d940a63)